### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by aaronshaf** [RangerMauve/dat-pub#1]
I love the term "pub" to describe a service where Dat archives can meet each other. Would be great to have a logo for this. Something like a beer glass?

**Comment by RangerMauve** [comment]
I haven't really gotten to a point whre this can be released, so I think a Logo might be too eary. :P

Would you be interested in becoming a collaborator in the repo to help move it forward?

I'm currently focused on [discovery-swarm-stream](https://github.com/RangerMauve/discovery-swarm-stream) so this has been put into my backlog.

Also, this becomes less necessary when paired with the new [dat peers sessionData](https://github.com/beakerbrowser/beaker/pull/1039) API being introduced in Beaker.

With the `sesionData` API you could advertise some piece of information to anyone replicating a Dat archive with you and thus allow peers to discover you throughout the network.

**Comment by aaronshaf** [comment]
RE: collaborator. Yes. 👍 

RE: `datPeers`. If a Dat Archive owner/writer goes offline, it seems a whisper(?) network of a record of recent/historical peers (and their respective archives) would need to be stored.

**Comment by RangerMauve** [comment]
I was thinking a discovery mechanism would look like this:

- Users create **their** dat for stroring applicaion data they want others to discovery
- Users set their sessionData to their dat URL
- Users look at their peers' sessionData
- They take their peer URLs and start indexing them or whatever
- Store their peer URL as a file in their `/.discovery` folder
- Indexing peers looks into their `/.discovery` folder
- Eventually everyone has a `/.discovery` folder that has everyone else's URLs

It doesn't matter who's online since it's an eventually-consistant gossip-based protocol. :D

**Comment by RangerMauve** [comment]
Ping @pfrazee

**Comment by pfrazee** [comment]
@RangerMauve yeah that definitely can work. You'll have no certainty about *who* the discovered dats are, but in the short term that might not matter.

**Comment by RangerMauve** [comment]
@pfrazee Yeah, I think identity could be determined by applications that build on top of this. Say, the dat contains some sort of data proving that it's tied to a given identity (like IndieAuth). It'd be easy to add a filter in to make sure a dat meets some sort of criteria before it's saved to discovery.

**Comment by munadikiehl** [comment]
Dear Sir
Hi, my name is Munadi Kiehl. I'm a graphic designer and I would like to collaborate on your open source project and to propose free logo design. If it's something you're interested in, please, let me know.
Best Regard,
Munadi Kiehl
